force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        String hello = null;
+        if ( x > y ) {
+            hello= 'Hello World!';
+        }
+        return hello; // 
```

Comment:
Good solution here.  You can skip the variable assigment. I dont use them if all I am doing is returning the value. 

if(x > y) return 'Hello World!'